internal/tiltfile/k8s.go

```diff
@@ -861,18 +861,19 @@ func convertPortForwards(val starlark.Value) ([]model.PortForward, error) {
 
 func (s *tiltfileState) portForward(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
 	var local, container int
-	var name string
+	var name, path string
 
 	// TODO: can specify host (see `stringToPortForward` for host validation logic)
 	if err := s.unpackArgs(fn.Name(), args, kwargs,
 		"local_port", &local,
 		"container_port?", &container,
-		"name?", &name); err != nil {
+		"name?", &name,
+		"path?", &path); err != nil {
```

Comment:
Hm, I was surprised we can only have one link, but in a few minutes I haven't come up with a good way to fit multiple links into this API. Bluh.

Reply:
you mean like there's no good way to tell Tilt to display links to `localhost:800/foo` and `localhost:8000/bar` just with the port forward syntax? Yeah :-/ i think the workaround for that (which I hadn't considered but that you brought up in #3857 is
```
k8s_resource(...,
  port_forwards=port_forward(8000, path='foo'),
  links('localhost:8000/bar')
)
```

hopefully we won't have too many folks wanting that?

Reply:
hm, reading that, it's not clear what `path` means - what do you think of `link_path`?

Reply:
> hopefully we won't have too many folks wanting that?

I've definitely wanted this on a lot of services I've worked on. Maybe a good answer is to make non-absolute `links` relative to the port forward.

Reply:
boooo yah you're right. will do.

Reply:
FYI i called it `link_path` in the tiltfile and left it as `path` internally but i'm happy for you to tell me that that's dumb